src/Cms/ModelWithContent.php

```diff
@@ -304,15 +304,6 @@ public function increment(
 		return $this->update([$field => $value]);
 	}
 
-	/**
-	 * Checks if the model is locked for the current user
-	 */
-	public function isLocked(): bool
-	{
-		$lock = $this->lock();
-		return $lock && $lock->isLocked() === true;
```

Comment:
`Comment`: Breaking change but I don't think many would use this at all. If we think this is an issue, we can add it as deprecated alias method pointing to `$this->panel()->lock()['isLocked']`.